.github/workflows/cicd.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

Comment:
I think it may be useful to changes this to `branches: [ master, development ]`, to run CI on `feature` PRs (open against `development`), and `release` PRs (open against `master`). Assuming, that we kept `master` and not `main`.

Reply:
+1 for master